app/components/provider_interface/user_permission_summary_component.rb

```diff
@@ -0,0 +1,28 @@
+module ProviderInterface
+  class UserPermissionSummaryComponent < ViewComponent::Base
+    PERMISSIONS = { manage_users: 'Manage users',
+                    manage_organisations: 'Manage organisation permissions',
+                    set_up_interviews: 'Set up interviews',
+                    make_decisions: 'Make offers and reject applications',
+                    view_safeguarding_information: 'View criminal conviction and professional misconduct',
+                    view_diversity_information: 'View sex, disability and ethnicity information' }.freeze
```

Comment:
Not sure on the best way to do this, but can we put these strings into the locale files? The organisation permission ones are already there, in `provider_relationship_permissions.yml`, but maybe we can extract them into a more generic place and include the user level permissions too?

Reply:
I've introduced a `user_permissions.yml` with the relevant descriptions.

Reply:
Can we remove this hash now then?